### ISSUE TITLE
Continuous Monitoring on RPi

### ISSUE THREAD

**Issue by yashasaxena** [yashasaxena/Melanoma_JSY_RPi#7]
Hi,

So I'm thinking of having a directory affiliated with a USB stick that I would insert on the RPi
be continuously monitored to see if there are any additional files/changes. 

Would this watchdog package be the way to do this? 
https://www.michaelcho.me/article/using-pythons-watchdog-to-monitor-changes-to-a-directory

And would the file name be (for this example) event.src_path

Thanks for your help!

**Comment by mlp6** [comment]
Yep  - the `watchdog` is perfect for the scenario where you would like to automatically trigger action based on the creation of a file in a specified directory.  One condition to check is that the files that the watchdog observer "sees" are not partially written, but only made available when fully flushed from memory caches.

**Comment by yashasaxena** [comment]
Ok great thanks! I'll look into the documentation to make sure I use the right checks
